src/lae_site/handlers/create_subscription.py

```diff
@@ -84,73 +76,73 @@ def handle_stripe_create_customer_errors(self, trace_back, error, details, email
         send_plain_email('info@leastauthority.com', 'support@leastauthority.com', body, headers)
         raise RenderErrorDetailsForBrowser(details)
 
-    def create_customer(self, stripe_authorization_token, user_email):
+    def create_customer(self, stripe_authorization_token, user_email, request):
         try:
             return self._stripe.create(
                 authorization_token=stripe_authorization_token,
                 plan_id=PLAN_ID,
                 email=user_email,
             )
         except stripe.CardError as e:
+            # Always return 402 on card errors
+            request.setResponseCode(402)
```

Comment:
You can get a symbolic constant for 402 from `twisted.web.http`: `PAYMENT_REQUIRED`

Reply:
oh cool stuff!